demo-app/client/README.md

```diff
@@ -0,0 +1,27 @@
+# Client
```

Comment:
What do you think about removing files we do not actively use? E.g. does this one bring any value? I do not have strict opinion, but in general prefer minimalistic approach.

Reply:
removed